src/node/oscillator.rs

```diff
@@ -272,13 +272,15 @@ impl OscillatorNode {
     ///
     /// * `context` - The AudioContext
     /// * `options` - The Oscillatoroptions
-    pub fn new<C: AsBaseAudioContext>(context: &C, options: OscillatorOptions) -> Self {
+    pub fn new<C: AsBaseAudioContext>(context: &C, options: Option<OscillatorOptions>) -> Self {
```

Comment:
nitpick: Ideally we only take one level of `Option<>` so maybe make this argument mandatory en let the members still be optional

Reply:
I think we should keep the option optional cause:
- compliance reason: this interface follows the web audio api interface
- Ergonomic reason: `options: None `vs `options: Options { field1: None, field2: None}`

The only drawback i see is that we have to deal with the option during construction.

Reply:
It's fine for now, later we can have a look at alle node options and figure out the best way.
For ergonomics you could use `options: Default::default()`